renovate.json

```diff
@@ -1,14 +1,27 @@
 {
   "extends": [
-    "config:base",
-    "schedule:earlyMondays"
+    "config:base"
   ],
   "automerge": true,
   "prHourlyLimit": 0,
+  "schedule": [
+    "before 10am on Monday"
```

Comment:
Maybe sunday just because it can take a while? IDK.

Reply:
I'm hoping with patches collapsed it won't be so bad.  Would rather it not be opening PRs on sunday to tempt people into looking 😄